.vsts-ci.yml

```diff
@@ -174,6 +175,7 @@ extends:
             publishTaskPrefix: 1ES.
           runtimeSourceProperties: /p:DotNetRuntimeSourceFeed=https://dotnetbuilds.blob.core.windows.net/internal /p:DotNetRuntimeSourceFeedKey=$(dotnetbuilds-internal-container-read-token-base64)
           ${{ if and(eq(parameters.runTestBuild, false), ne(variables['Build.Reason'], 'PullRequest')) }}:
+            timeoutInMinutes: 90
```

Comment:
Would it make sense to define these as variables or constants, something like `sourceBuildTimeoutInMinutes`, makes it a bit easier than processing the build reason logic each time. and gives a central place to configure all the timeouts

Reply:
Yup! That would make a ton of sense! I'd love to do that if it were possible. 😅

This value is required to be a compile-time constant. There's no mechanism to centrally define parameters either. You can create either pipeline parameters or template parameters and reuse them. There's no concept of a "constant" in the way you're thinking. I used a template parameter for the `dotnet-format-integration.yml`. In this situation, your only option is to define a pipeline parameter, which always shows up in the UI when manually running a job. I avoid adding pipeline parameters unless they are intended to be settable by the user in the UI. If only there was a way to set pipeline parameters to 'hidden' from the UI. Oh well. 😢